### DISCUSSION TITLE
Question: Integration with QuickAdd Macros

### DISCUSSION THREAD

**Discussion by raven2cz** [chrisgurney/obsidian-note-toolbar#58] [Q&A] [answered]
Is it possible to combine your plugin with the capability of running macros assigned in QuickAdd plugins? I have quite a few and would like to use some via a button. Thank you for the advice.

**Comment by chrisgurney** [comment, marked as answer]
Is it possible for you to use something like the Advanced URI plugin to execute your macros?

See: https://github.com/chhoumann/quickadd/discussions/451#discussioncomment-5554442

And: https://github.com/chrisgurney/obsidian-note-toolbar/wiki/How-to-get-a-URI-with-Advanced-URI

If so, you can then use the URI in a URI toolbar item.

Let me know if that helps!

**Comment by raven2cz** [comment]
Works! 
Thank you very much for the quick and perfect response. I really like your Obsidian plugin. Once I've learned how to use it properly, I'll be very happy to write a review. Thanks.

**Comment by FelipeRearden** [comment]
Additionally to @chrisgurney approach: you can use a QuickAdd choice as a ordinary command using the `Command` type on Note Toolbar.

![IMG_1170](https://github.com/chrisgurney/obsidian-note-toolbar/assets/85180696/5303a7e5-79be-4c99-84f1-ab0a63937e2e)


To do this, on QuickAdd choices list you have to enable the `thunder` icon on right of every choice that you have. 

![IMG_1171](https://github.com/chrisgurney/obsidian-note-toolbar/assets/85180696/d53baf49-de60-4efd-8736-0c2ec2381e7e)


This way you can run a QuickAdd Macro as an obsidian command from command palette `QuickAdd: your macro name here`
